soc/arm/nanoxplore_ng/ngultra/Kconfig.defconfig.ngultra

```diff
@@ -9,4 +9,8 @@ config SOC
 config NUM_IRQS
 	default 192
 
+config UART_NX_NGULTRA
+	default y
+	depends on SERIAL
+
```

Comment:
No need for this if you add the `depends on DTS_HAS_*_ENABLED` and `default y` in the driver's Kconfig.

Reply:
Done